### PR TITLE
feat: ckBTC min_confirmations renamed to minConfirmations

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

As discussed in #701, let's rename `min_confirmations` parameter of `getUtxos` in ckBTC library to `minConfirmations`.

# Changes

- Parameter and mapping updated accordingly.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 - Rename values of enum Topic and NnsFunction to match the backend values.
 - Use different request/response types for NNS Governance proposals, and different fields for `InstallCode` proposals.
+- The `getUtxos` parameter `filter.min_confirmations` has been renamed to `filter.minConfirmations` for consistency with the general naming conventions used in `@dfinity/ckbtc`.
```
Comment (AntonioVentilii-DFINITY) [inline CHANGELOG.md:7]: What is the usual flow for breaking changes? Should it goes maybe in a separate Section, above Features? Just to be more evident

**Review by AntonioVentilii-DFINITY — COMMENTED** [review]
In the end, I left `min_confirmations` in `getBalance` too, just to be consistent, thinking that we would change everything together in another PR. Shall we change it here too? Or separate PR?

File: CHANGELOG.md (hunk 1 above)
Comment (peterpeterparker) [reply CHANGELOG.md:7]: This section is "Breaking Changes" or am I missing something?

**Comment by peterpeterparker** [comment]
> In the end, I left min_confirmations in getBalance too, just to be consistent, thinking that we would change everything together in another PR. Shall we change it here too? Or separate PR?

@AntonioVentilii-DFINITY oh! yeah in another PR. No need to add anything in CHANGELOG as it is not yet released but for separation of concern, let's split in another PR. Works for you?

File: CHANGELOG.md (hunk 1 above)
Comment (AntonioVentilii-DFINITY) [reply CHANGELOG.md:7]: oh gah... very sorry, I misread

File: packages/ckbtc/src/bitcoin.canister.spec.ts [hunk 2]
```diff
@@ -79,6 +79,55 @@ describe("BitcoinCanister", () => {
...
+        });
+
+        const { filter, ...rest } = params;
+        const page = [1, 2, 3];
+        const pageParams: Omit<GetUtxosParams, "certified"> = {
+          ...rest,
+          filter: {
+            page,
+          },
+        };
```
[the remark this answers is not in the document]
Comment (peterpeterparker) [reply packages/ckbtc/src/bitcoin.canister.spec.ts:117]: Ah yeah you are right, I was more doing it for readability but let me change that.

[the remark this answers is not in the document]
Comment (peterpeterparker) [reply packages/ckbtc/src/bitcoin.canister.spec.ts:116]: Done you are right, better. Thanks.

**Pull request merged**